awsenergylabelerlib/entities.py

```diff
@@ -813,11 +813,11 @@ def _export_to_s3(self, s3_url, filename, data):
         s3 = boto3.client('s3')  # pylint: disable=invalid-name
         parsed_url = urlparse(s3_url)
         bucket_name = parsed_url.netloc
-        dst_path = parsed_url.path
+        dst_path = f"{parsed_url.path}{datetime.now().year}/{datetime.now().month}/{datetime.now().day}/"
```

Comment:
`
>>> s3_url="s3://bucketname/teamname/"
>>> parsed_url = urlparse(s3_url)
>>> dst_path = parsed_url.path
>>> print(dst_path)
/teamname/
>>> dst_path = f"{parsed_url.path}{datetime.now().year}/{datetime.now().month}/{datetime.now().day}/"
>>> print(dst_path)
/teamname/2022/5/17/
`

As you can see, `parsed_url.path` has both a leading and trailing slash. Hence, no slash is needed between `{parsed_url.path}{datetime.now().year}`

The `.lstrip("/")` strips away only the leading (first) slash:
`
>>> filename="labeledaccount.json"
>>> dst_filename = urljoin(dst_path, filename)
>>> print(dst_filename)
/teamname/2022/5/17/labeledaccount.json
>>> dst_filename = urljoin(dst_path, filename).lstrip("/")
>>> print(dst_filename)
teamname/2022/5/17/labeledaccount.json
`

This leading slash needs to be removed, because `s3.upload_file(temp_file.name, bucket_name, dst_filename)` adds a slash after the bucket name which leads to `s3://bucketname//teamname/2022/05/17/labeledaccount.json`. From experience I know this won't work with the double slash.